pkgs/swiftgen/swift2objc/lib/src/parser/_core/utils.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:convert';
+import 'dart:io';
+
+import '../../ast/_core/interfaces/declaration.dart';
+import 'parsed_symbol.dart';
+
+typedef DeclarationsMap = Map<String, Declaration>;
+typedef ParsedSymbolsMap = Map<String, ParsedSymbol>;
+typedef JsonMap = Map<String, dynamic>;
+
+typedef ParserFunction = Declaration Function(
```

Comment:
What's this for?

Reply:
I was trying to unify the signature of parsers but then realized it's not needed. I just forgot to remove it.